README.md

```diff
@@ -1 +1,18 @@
-# hw01
\ No newline at end of file
+# HOMEWORK ONE   
+## Who is Paulino?
+![](https://github.com/pdiazm/hw01/blob/master/PD_Headshot.jpg)
```

Comment:
It is great that you accessed the photo through your repo! It would have been better if I had tried doing the same as well.